bin.js

```diff
@@ -16,18 +17,35 @@ if (newRules.length) {
   process.exit(1)
 }
 
-function getConfig() {
+function getConfigFile() {
   var specifiedFile = process.argv[2]
   if (specifiedFile) {
     // this is being called like: eslint-find-new-rules eslint-config-mgol
     if (isAbsolute(specifiedFile)) {
-      return require(specifiedFile)
+      return specifiedFile
     } else {
-      return require(path.join(process.cwd(), specifiedFile))
+      return path.join(process.cwd(), specifiedFile)
     }
   } else {
     // this is not being called with an arg. Use the package.json `main`
```

Comment:
Well, I'm currently about to rewrite that part anyway (to restore the desired behavior described in the comment).